src/App.js

```diff
@@ -1,17 +1,24 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
-import './App.css'
 import 'bootstrap/dist/css/bootstrap.min.css'
+import Welcome from './scenes/Login/Welcome'
+import './App.css'
+import Login from './scenes/Login/Login'
 import Home from './scenes/Home/Home'
 import WeatherHourlyList from './scenes/WeatherHourlyList/WeatherHourlyList'
 
 function App () {
+  const [foundUser, setFoundUser] = useState()
+  if (!foundUser) {
+    return <Login setFoundUser={setFoundUser} />
+  }
```

Comment:
No need to save user info right now. Can remove it.

Reply:
yes , Done

Reply:
Missing this ☝️

Reply:
so sr, i forgot App component
![image](https://user-images.githubusercontent.com/62316127/123151337-f50d4500-d428-11eb-9c38-d824ea51e498.png)

Reply:
![image](https://user-images.githubusercontent.com/62316127/123399300-13c72500-d56a-11eb-9f47-8a7312ba2f68.png)